client/state/automated-transfer/reducer.js

```diff
@@ -6,18 +6,30 @@ import { combineReducers } from 'redux';
 /**
  * Internal dependencies
  */
-import { AUTOMATED_TRANSFER_STATUS_SET } from 'state/action-types';
-import { automatedTransferStatusSchema } from './schema';
-import { createReducer } from 'state/utils';
+import eligibility from './eligibility/reducer';
+import {
+	keyedReducer,
+	withSchemaValidation,
+} from 'state/utils';
+import { AUTOMATED_TRANSFER_STATUS as transferStatus } from './constants';
+import { automatedTransfer as schema } from './schema';
+import {
+	AUTOMATED_TRANSFER_STATUS_SET as SET_STATUS,
+} from 'state/action-types';
 
-// Stores the status of automated transfers per site
-export const status = createReducer( {}, {
-	[ AUTOMATED_TRANSFER_STATUS_SET ]: ( state, action ) => ( {
-		...state,
-		[ action.siteId ]: action.automatedTransferStatus,
-	} ),
-}, automatedTransferStatusSchema );
+export const status = ( state = transferStatus.ALOOF, action ) =>
```

Comment:
Beware of this!
`PluginAutomatedTransfer` [renders whenever the status exists](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plugins/plugin-automated-transfer/index.jsx#L67-L69)!

We could solve this by only checking against a subset of that constant object (eg. `START, SETUP, CONFLICTS, COMPLETE` - as `LEAVING` is now an inner state, so we could exclude it).

Or, maybe better yet rewrite the mapping:
`ALOOF` or falsy value = not rendered
`SETUP, CONFLICTS, COMPLETE` = as is now
`START` = default value (so in this new scenario, it would also include `INQUIRING`)